HopsanGUI/Dialogs/OptimizationDialog.cpp

```diff
@@ -568,16 +620,53 @@ void OptimizationDialog::applyParameters()
 }
 
 
```

Comment:
Why is this comparison needed, is it not enough that the signal is emitted, (textChanged)?
Or is that signal emitted to often?